recipes/rpbasicdesign/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "rpbasicdesign" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "2c3123bbb2daabba3f7e21ca6896a2bf851b1c2ce1dbb446ab707a198a9853c0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  number: 0
  noarch: python
```